pkgs/test/texlive/default.nix

```diff
@@ -41,7 +41,7 @@
       \end{document}
     '';
   } ''
-    chktex -v -nall -w1 "$input" 2>&1 | tee "$out"
+    (chktex -v -nall -w1 "$input" 2>&1 || true) | tee "$out"
```

Comment:
Does this make the test succeed, no matter what? What are we really testing here? If there is an expected exit value, we should rather check for that, and document what it means.

Reply:
the test only passes if the grep on the line below also returns 0, i.e. the output of chktex contains the string "One warning printed". This is in my opinion a reasonable interpretation of "chktex does the right thing", but also checking the return code is better.